core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -108,9 +108,8 @@
                                   [showOpportunityButton]="false">
         </oppia-opportunities-list>
       </div>
-      <div *ngIf="isAccomplishmentsTab && accomplismentsEnabled && activeAccomplishmentType === 'stats'">
-        <contributor-stats
-                            [type]="defaultContributionType">
+      <div *ngIf="activeTabIsAccomplishments && accomplishmentsTabIsEnabled && activeAccomplishmentType === 'stats'">
```

Comment:
In these composite && statements, always start with the accomplishmentsTabIsEnabled check first because it determines whether we can do the thing at all or not. If this is false then everything else doesn't matter.